lib/pubsub/subscription.js

```diff
@@ -368,13 +363,17 @@ Subscription.prototype.ack = function(ackIds, callback) {
 
   callback = callback || util.noop;
 
-  this.request({
-    method: 'POST',
-    uri: ':acknowledge',
-    json: {
-      ackIds: ackIds
-    }
-  }, function(err, resp) {
+  var protoOpts = {
+    service: 'Subscriber',
```

Comment:
I think if we moved to extending a `ProtoServiceObject` class, we could maybe localize some of these options in the constructor?

```js
this.protoService = 'Subsriber';
```

..?